src/include/dns-server/dialog-main.rb

```diff
@@ -5,12 +5,15 @@
 # Summary:	Data for configuration of dns-server, input and output functions.
 # Authors:	Jiri Srain <jsrain@suse.cz>
 
-require "ui/service_status"
+require "dns-server/service_widget_helpers"
 
 module Yast
   # Representation of the configuration of dns-server.
   # Input and output routines.
   module DnsServerDialogMainInclude
+    include Yast::Logger
```

Comment:
I guess it is necessary to require yast to use the logger. Probably it works because another file is already requiring yast.

Reply:
So, I must to explicitly `require "yast"` at the beginning, isn't it?

Reply:
requiring `yast` is quite common.